source/common/runtime/runtime_features.cc

```diff
@@ -51,6 +51,7 @@ RUNTIME_GUARD(envoy_reloadable_features_http1_allow_codec_error_response_after_1
 RUNTIME_GUARD(envoy_reloadable_features_http1_connection_close_header_in_redirect);
 RUNTIME_GUARD(envoy_reloadable_features_http1_use_balsa_parser);
 RUNTIME_GUARD(envoy_reloadable_features_http2_decode_metadata_with_quiche);
+RUNTIME_GUARD(envoy_reloadable_features_http2_use_oghttp2);
```

Comment:
oops sorry this should have a release note in changelogs/current.yaml, letting folks know the h2 framer is changing and they can revert by flipping back